intel-retail/automated-self-checkout
# fix: resolve the issue of the current model downloader

current getModel.sh script and other model downloading related scripts are all using relative path like . or .., which is not working when the scripts are
  running in different directories.
  This PR resolves the issue of using relative path and uses the absolute path to run via finding out the execution path on the runtime.

  Fixes: #96

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**96**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

1. unit tests: run both unit test scripts under download_models/ directory, they should all pass
2. test it with ./docker-run.sh:   `sudo ./docker-run.sh`:
```
$ ./camera-simulator/camera-simulator.sh
$ sudo ./docker-run.sh --workload opencv-ovms --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --ocr_disabled --barcode_disabled --classification_disabled
```
you should see the models are now downloading.

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
